components/icons/ConfettiIcon.tsx

```diff
@@ -10,7 +10,7 @@ const ConfettiIcon = ({ className }: Props) => {
       width={24}
       height={24}
       viewBox="0 0 24 24"
-      stroke-width={2}
+      strokeWidth={2}
```

Comment:
to fix a small warning 
![image](https://user-images.githubusercontent.com/6569308/216749537-7ee7c27e-e89c-44f6-9966-7e9543e61dda.png)